python-shell/src/gafferpy/gaffer_operations.py

```diff
@@ -2633,299 +665,163 @@ def to_json(self):
+        return operation_json
 
+class ToVertices(ToVertices):
     def to_json(self):
-        operation = super().to_json()
-
-        if self.variable_name is not None:
-            operation['variableName'] = self.variable_name
-
-        return operation
-
-
-class GetVariables(Operation):
-    CLASS = 'uk.gov.gchq.gaffer.operation.impl.GetVariables'
+        operation_json = super().to_json()
+        if self.input is not None:
+            json_seeds = []
+            if not isinstance(self.input, list):
+                self.input = [self.input]
+            for seed in self.input:
+                if isinstance(seed, ElementSeed):
+                    json_seeds.append(seed.to_json())
+                else:
+                    json_seeds.append(EntitySeed(seed).to_json())
+            operation_json['input'] = json_seeds
+        return operation_json
 
-    def __init__(self, variable_names=None, options=None):
-        super().__init__(_class_name=self.CLASS, options=options)
-        self.variable_names = variable_names
+# Entity Input
 
+class GetWalks(GetWalks):
     def to_json(self):
-        operation = super().to_json()
-
-        if self.variable_names is not None:
-            operation['variableNames'] = self.variable_names
-
-        return operation
-
-
-class SetVariable(Operation):
-    CLASS = 'uk.gov.gchq.gaffer.operation.impl.SetVariable'
-
-    def __init__(self, input=None, variable_name=None, options=None):
-        super().__init__(_class_name=self.CLASS, options=options)
-        self.input = input
-        self.variable_name = variable_name
+        operation_json = super().to_json()
+        if self.input is not None:
+            json_seeds = []
+            if not isinstance(self.input, list):
+                self.input = [self.input]
+            for seed in self.input:
+                if isinstance(seed, EntitySeed):
+                    json_seeds.append(seed.to_json())
+                else:
+                    json_seeds.append(EntitySeed(seed).to_json())
+            operation_json['input'] = json_seeds
+        return operation_json
 
+class GetAdjacentIds(GetAdjacentIds):
     def to_json(self):
-        operation = super().to_json()
-
-        if self.variable_name is not None:
-            operation['variableName'] = self.variable_name
-
+        operation_json = super().to_json()
         if self.input is not None:
-            operation['input'] = self.input
+            json_seeds = []
+            if not isinstance(self.input, list):
+                self.input = [self.input]
+            for seed in self.input:
+                if isinstance(seed, EntitySeed):
+                    json_seeds.append(seed.to_json())
+                else:
+                    json_seeds.append(EntitySeed(seed).to_json())
+            operation_json['input'] = json_seeds
+        return operation_json
 
-        return operation
+class GetElementsWithinSet(GetElementsWithinSet):
```

Comment:
The GitHub diff makes these difficult to read, but at the bottom of this file here, when you see a Class overriding itself, it is because some extra helper functionality has been manually added to the generated class to retain backwards compatibility.
In this example, we are wrapping any non-list input into a list, then wrapping plain vertices in EntitySeeds.

Perhaps some patterns (like an input of seeds) could be added directly to the generating code in fishbowl to eliminate the need for as many manual additions. As well as cleaning this class up, it would mean when users generate their own versions of gafferpy, they get to keep these extensions without needing to extend directly from this file.